tool/common/print.go

```diff
@@ -30,6 +30,11 @@ func PrintError(err error) {
 	color.Red("[ERROR]: %v\n", trace.UserMessage(err))
 }
 
+// PrintWarn outputs a warning message to stdout.
+func PrintWarn(message string, format ...interface{}) {
```

Comment:
nit: `format` would be expceted to be the format string with placeholders - this looks like a slice of arguments, so `args`?